MagicalRecord/Categories/DataImport/NSEntityDescription+MagicalDataImport.h

```diff
@@ -7,11 +7,12 @@
 //
 
 #import <CoreData/CoreData.h>
+#import "MagicalRecordXcode7CompatibilityMacros.h"
```

Comment:
The user will eventually get a "header not found" compiler error.
Or he will have to specify some search path. Which means that the framework does not work out of box.

So, it's not just about umbrella headers. It's about any public header in your framework.